src/app/components/about/about.component.ts

```diff
@@ -13,21 +14,23 @@ export class AboutComponent implements OnInit, OnDestroy {
     raidenVersion = '';
     networkName = '';
 
-    private subscription: Subscription;
+    private ngUnsubscribe = new Subject();
```

Comment:
Really nice approach. I have never thought that :)

Reply:
The issue with the previous approach was that when the observable of the main subscription completes the sub-subscriptions would be unsubscribed. I had a failing test due to that